models/shoppingCartedProduct.js

```diff
@@ -1,4 +1,8 @@
 module.exports = (sequelize, DataTypes) =>
   sequelize.define('shopping_carted_product', {
     quantity: DataTypes.INTEGER,
```

Comment:
because shopping-carted products [sic] each have productId, there is no need to give them the title, description, or unitPrice, which won't differ from those corresponding properties on the product. it remains necessary to give them a quantity property since those properties generally will differ from the quantity property on the product.